sourcegraph/sourcegraph
# JetBrains: display initial sign in with sourcegraph panel

Closes #56024 

Add initial panel to login with sourcegraph.com or enterprise instance when there are no accounts defined

Sign in for free with Sourcegraph.com doesn't work yet, it should be implemented after https://github.com/sourcegraph/sourcegraph/issues/56388



<img width="491" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/7345368/61225786-2ba4-44e5-98f1-aaf65045c596">

## Test plan
- Open project with cody installed when you don't have any accounts then panel with onboarding information should appear
- Sign in with an Enterprise instance should display login panel and added account should be marked as default
